### DISCUSSION TITLE
Access denied when drop tables

### DISCUSSION THREAD

**Discussion by gowdra01** [trinodb/trino#15920] [Q&A]
Hi , I am using trinodb in k8s and connected to hive.. but unfortunately I am getting Access Denied error when I trying to alter table to add columns ... I have added the rules.json and hive-secutiry for the user which logged in via ldap.. any idea why is that error ?

I can do show tables; or select query it work but not add columns

```yaml
rules.json: |-
  {
    "catalogs": [
      {
        "catalog": "hive",
        "allow": "all"
      }
    ],
    "queries": [
      {
        "user": "svc-presto",
        "allow": ["execute", "kill", "view"]
      }
    ]
  }
 hive-security.properties: |-
   {
     "schemas": [
       {
         "schema": ".*",
         "owner": true
       }
     ],
     "tables": [
       {
         "privileges": ["SELECT", "ALTER","DROP", "INSERT", "DELETE", "OWNERSHIP"]
       }
     ]
   }
```

any idea why this is happening ?

**Comment by user-GitHub-user** [comment]
see: https://trino.io/docs/current/connector/hive-security.html
try config:
hive.allow-drop-table, hive.allow-rename-table, hive.allow-add-column, hive.allow-drop-column and hive.allow-rename-column
